apps/web/src/services/advanced/composables/querys/use-workspace-group-user-find-query.ts

```diff
@@ -0,0 +1,38 @@
+import type { ComputedRef } from 'vue';
+import { computed } from 'vue';
+
+import { useWorkspaceGroupUserApi } from '@/api-clients/identity/workspace-group-user/composables/use-workspace-group-user-api';
+import type { WorkspaceGroupUserFindParameters } from '@/api-clients/identity/workspace-group-user/schema/api-verbs/find';
+import { useServiceQueryKey } from '@/query/core/query-key/use-service-query-key';
+import { useScopedQuery } from '@/query/service-query/use-scoped-query';
+
+import ErrorHandler from '@/common/composables/error/errorHandler';
+
+interface UseWorkspaceGroupUserFindQueryOptions {
+    params: ComputedRef<WorkspaceGroupUserFindParameters>;
+    enabled?: ComputedRef<boolean>;
+}
+
+export const useWorkspaceGroupUserFindQuery = ({ params, enabled }: UseWorkspaceGroupUserFindQueryOptions) => {
+    if (!params.value.workspace_group_id) {
+        ErrorHandler.handleError(new Error('workspace_group_id is required'));
+    }
```

Comment:
I think, this may not work as you expect.
This code is written in out of vue reactivity area, so even if params change, this code will not be re-executed.

Suggestion:
how about writing this code in `queryFn`?

```
...
queryFn: () => {
    if (!params.value.workspace_group_id) {
        throw new Error('workspace_group_id is required');
    }
    return workspaceGroupUserAPI.find(workspaceGroupUserFindQueryParams.value),
}
```